Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.yml

```diff
@@ -1,6 +1,6 @@
 sectionOrder:
-- Connect
-- Collect
+  - Connect
```

Comment:
Please revert all changes that are not related to your change and let me know so I can review this file.

Reply:
reverted